File: app/src/main/java/com/sliide/news/core/viewmodel/ImageViewBinding.java
package com.sliide.news.core.viewmodel;

import android.widget.ImageView;

import androidx.databinding.BindingAdapter;

import com.squareup.picasso.Picasso;

public class ImageViewBinding {

    @BindingAdapter("imageUrl")
    public static void loadImage(ImageView view, String imageUrl) {
        Picasso.with(view.getContext())
                .load(imageUrl)
                .fit()
                .into(view);
    }
}

File: app/src/main/java/com/sliide/news/core/viewmodel/ImageViewBinding.kt
package com.sliide.news.core.viewmodel

import android.widget.ImageView
import androidx.databinding.BindingAdapter
import com.squareup.picasso.Picasso

object ImageViewBinding {
    @JvmStatic
    @BindingAdapter("imageUrl")
    fun loadImage(view: ImageView, imageUrl: String) {
        Picasso.get()
            .load(imageUrl)
            .fit()
            .into(view)
    }
}
